src/lib.rs

```diff
@@ -441,26 +353,25 @@ impl Default for CosmicEditSpriteBundle {
             visibility: Visibility::Hidden,
             computed_visibility: Default::default(),
             background_color: Default::default(),
-            editor: Default::default(),
             text_position: Default::default(),
             cosmic_metrics: Default::default(),
-            cosmic_edit_history: Default::default(),
             cosmic_attrs: Default::default(),
             background_image: Default::default(),
             max_lines: Default::default(),
             max_chars: Default::default(),
+            text: Default::default(),
         }
     }
 }
 
 #[derive(Clone)]
-pub struct EditHistoryItem {
```

Comment:
`velo` uses it here: https://github.com/StaffEngineer/velo/blob/b75fa7c770ed5bb78652dcb508382c64152d7e4b/src/ui_plugin/systems/keyboard.rs#L163

Reply:
Yeah I can now think of plenty of reasons to want to programatically access and control history, had tunnel vision when I made that private. Reverted